modules/nixos/sddm.nix

```
@@ -55,5 +56,12 @@ in
           ;
       })
     ];
+
+    assertions = [
+      {
+        assertion = config.services.displayManager.sddm.package != pkgs.kdePackages.sddm;
+        message = "Not using the kdePackages.sddm package, will cause the theme to fail being applied";
+      }
+    ];
```

```suggestion
    assertions = lib.optional cfg.assertQt6Sddm {
      assertion = config.services.displayManager.sddm.package != pkgs.kdePackages.sddm;
      message = "Not using the kdePackages.sddm package, will cause the theme to fail being applied";
    };
```

`assertQt6Sddm` should probably be
```nix
{
  assertQt6Sddm = lib.mkEnableOption "checking `services.displayManager.sddm.package` is the Qt 6 version. This is to ensure the theme is applied properly, but may have false positives in the case of overridden packages for example" // { default = true; };
}
```
(hopefully with better formatting :trollface:)

---

wow that's cursed but sure.